### PR TITLE
Fix back button

### PR DESCRIPTION
**Pull request by EvilGremlin**
Better than https://github.com/MarlinFirmware/Marlin/pull/24690 (whytf i added so many conditions in the first place? guess i didn't  notice IS_UNTIPANEL define for COLOR_UI.

Kinda improved filchange menu entry (restored previous insert for encoder users) but it's still so crude, i don't like it...

### CODE REVIEW

File: Marlin/src/lcd/menu/menu_main.cpp [hunk 1]
```diff
@@ -458,6 +459,17 @@ void menu_main() {
...
+  #if DISABLED(DISABLE_ENCODER) && ENABLED(ADVANCED_PAUSE_FEATURE)
+    #if E_STEPPERS == 1 && DISABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+      YESNO_ITEM(MSG_FILAMENTCHANGE,
+        menu_change_filament, nullptr,
+        GET_TEXT_F(MSG_FILAMENTCHANGE), (const char *)nullptr, F("?")
+      );
+    #else
+      SUBMENU(MSG_FILAMENTCHANGE, menu_change_filament);
+    #endif
+  #endif
```
Comment (thinkyhead) [inline Marlin/src/lcd/menu/menu_main.cpp:466]: I'm not fond of having repeated code that must be altered exactly in tandem with some identical block in another place, especially when no note is added to the code to alert developers of this situation. I will see about making this editable in a single location.

Comment (EvilGremlin) [reply Marlin/src/lcd/menu/menu_main.cpp:466]: I''m cringing at this too. If only we could use macro inside macro substitutes... 
Ideally, shed away non-advanced_pause method (and all non-UBL leveling), then have defeatureing inside them for 128k builds.

**Comment by sbaeder** [comment]
Just as an FYI - this also broke my SKR2 (new) with a standard REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER.  as mentioned, I did "figure it out" and added a hard-coding for the define...

Scott

**Comment by thisiskeithb** [comment]
Can this be merged. Pretty please 🙂

**Comment by EvilGremlin** [comment]
yeah i have no idea why Scott cleaned it up but didn't merge
On the other hand, we getting good statistic on how many bugfix branch users can figure out simple stuff :)

**Comment by thijstriemstra** [comment]
> Can this be merged. Pretty please

Same issue here; +1

**Comment by pmjdebruijn** [comment]
This change caused "Filament Change" to appear after "About this printer", which is very awkward (at least with MEDIA_MENU_AT_TOP enabled).

**Comment by thisiskeithb** [comment]
> which is very awkward

Agreed. Menu items probably need a scrub / revamp since the About Printer menu item can appear mid-menu instead of at the end or deeper in the menu tree.

**Comment by EvilGremlin** [comment]
iirc original idea of putting filchange at the bottom was to quick access it by scrolling up, but i think it got lost
At least i can't scroll menu up to last item, there should be "continous scrolling" option but i dont see it in current configs

**Comment by pmjdebruijn** [comment]
I'm also not so sure that quick-access is a good idea to begin with, because this ends up being fairly use-case specific.

**Comment by EvilGremlin** [comment]
yep, very specific, like every bowden ever

**Pull request merged**